test/integration/config/file-backend-encoded-slash-handling.json

```diff
@@ -0,0 +1,54 @@
+{

```

Comment:
This is the same config as `server-memory.json` but with a file backend right? Would just call it `server-file.json` then.